apps/heft/UPGRADING.md

```diff
@@ -141,8 +173,9 @@ The following is an example "heft.json" file defining both a "build" and a "test
           }
         },
         "copy-assets": {
-          "taskEvent": {
-            "eventKind": "copyFiles",
+          "taskPlugin": {
+            "packageName": "@rushstack/heft",
```

Comment:
We were talking about omitting this for built-in plugins.

Reply:
@iclanton I left this out of this PR intentionally. Right now, `pluginName` is optional, and defaults to the first plugin iff there is only one plugin contained in the plugin package. I think this makes sense, because the plugin name isn't terribly important other than to direct Heft to which plugin to load. And I also think there will be a lot of plugin packages with a single plugin, so this could slim down the config significantly. If we _also_ defaulted non-provided plugin packages to `@rushstack/heft`, then we would be in a situation where we would have to then enforce that the plugin name is always provided, but _only_ in the case where `@rushstack/heft` is the package. This seemed overly complex, and leaving it with the original rules (`pluginPackage` must be specified, `pluginName` must be specified if there is more than one plugin) feels much more straightforward

Reply:
There ended up being some potential ambiguity about defaults between "current package" vs "@rushstack/heft"; both sound like a reasonable expectation for default behavior, so might be clearer to not have a default.